.github/workflows/release.yml

```diff
@@ -88,21 +88,21 @@ jobs:
             - name: Push to npm
               run: |
                   yarn npm-publish
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft != true
 
             - name: Push to npm (dry run)
               run: |
                   yarn npm-publish --dry-run
-              env:
-                  DEVCYCLE_PROD_SLEUTH_API_TOKEN: ${{ secrets.SLEUTH_API_KEY }}
               if: inputs.draft == true
 
             - name: Build UMD bundles
               run: |
                   yarn nx build:cdn js
 
             - name: Upload CDN Release
-              run: aws s3 sync dist/sdk/cdn/js s3://js.devcycle.com/ --include "devcycle.min.js*" --acl public-read
+              run: |
+                JQ_PATH=".version"
+                SDK_VERSION="$(cat sdk/js/package.json | jq -r ${{ JQ_PATH }})"
+                aws s3 sync dist/sdk/cdn/js s3://js.devcycle.com/latest/ --include "devcycle.min.js*" --acl public-read
```

Comment:
hmm wait this changes the path the current CDN file is being uploaded on, that's gonna prevent everyone's current setups from receiving any more updates

Reply:
yeah, do we want this to be an explicit change over? that way current users aren't impacted

Reply:
but why wouldn't we want to keep uploading to the existing path with the latest version?

Reply:
will make the change to update the existing path as well 👍